libr/bin/format/pe/pe.c

```diff
@@ -795,17 +795,23 @@ int PE_(bin_pe_get_claimed_checksum)(struct PE_(r_bin_pe_obj_t)* bin) {
 }
 
 int PE_(bin_pe_get_actual_checksum)(struct PE_(r_bin_pe_obj_t)* bin) {
-	int i, j, checksum_offset = 0;
+	size_t i, j, checksum_offset = 0;
 	ut64 computed_cs = 0;
 	int remaining_bytes;
 	int shift;
 	ut32 cur;
 	if (!bin || !bin->nt_header_offset) {
 		return 0;
 	}
+	const size_t buf_sz = 0x1000;
```

Comment:
why  0x1000 ? can we use a PE constant here?

Reply:
@radare its just a value i got from testing, its just to avoid calling read() too frequently and creating an io bottleneck